client/app/cloud/project/compute/infrastructure/flavor.service.js

```diff
@@ -124,59 +127,59 @@
       augmentedFlavor.frequency = this.CLOUD_INSTANCE_CPU_FREQUENCY[flavor.type];
 
       if (/vps/.test(flavor.type)) {
-        augmentedFlavor.vps = true;
-        augmentedFlavor.diskType = 'ssd';
-        augmentedFlavor.flex = false;
-        augmentedFlavor.shortGroupName = flavor.name;
+        return _(augmentedFlavor)
```

Comment:
Implicit chaining isn't helpful in that case. You can simply return `return _.assign(…)`.

Reply:
Otherwise you could use [Object Spread Operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals) but it requires a babel plugin `"plugins": ["transform-object-rest-spread"]`.

ref: https://redux.js.org/recipes/usingobjectspreadoperator